utils/mentionStalker.js

```diff
@@ -9,6 +9,9 @@ const moment = require('moment')
 module.exports = (self, log, config) => {
   self.on('messageCreate', (msg) => {
     if (msg.author.id !== self.user.id && msg.channel.guild && ~msg.content.indexOf(self.user.id)) {
+      if (!config.mentionNotificator.logBlocked && self.relationships.has(msg.author.id)) {
```

Comment:
`config.mentionNotificator.logBlocked`. mentionNotificator is an object, you should place `logBlocked` inside that object instead of being in a new line on the config, and change its name to `logBlockedUsers`.